### PR TITLE
Make descriptive error message for new lesson page

### LINKED ISSUE #189: More Descriptive Error Messages on the Create Lesson Modal.
**Describe the bug**
When a user submits the form on the `Create Lesson` Modal without specifying a name, they don't receive a descriptive error message. We should update the API to return a descriptive message when there is missing data (check out the `Create Unit` Modal and `POST '/user'` endpoint for how the descriptive message should be returned).

### PR DESCRIPTION
**Pull request by rustom**
## Description
This creates a more specific error message on the new lesson page. Also just shifted up the close function a few lines. 

Fixes #189 

## Screenshots
IMG_1994

### CODE REVIEW

File: client/src/pages/CreateLesson/CreateLesson.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ import { createLesson } from 'api'
 import { useErrorWrap } from 'hooks'
 
 const CreateLesson = ({ navigation }) => {
+  // Closes the modal
```
Comment (ashayp22) [inline client/src/pages/CreateLesson/CreateLesson.js:12]: Any reason why the location of this method changed?

**Review by ashayp22 — COMMENTED** [review]
Looks great!

**Review by ashayp22 — APPROVED** [review]

File: client/src/pages/CreateLesson/CreateLesson.js (hunk 1 above)
Comment (rustom) [reply client/src/pages/CreateLesson/CreateLesson.js:12]: To match the CreateUnit

**Comment by rustom** [comment]
Realizing now that this PR will probably be defunct now that we have switched from error messages to greyed out submission buttons, feel free to delete

**Comment by ashayp22** [comment]
> Realizing now that this PR will probably be defunct now that we have switched from error messages to greyed out submission buttons, feel free to delete

Will go ahead and close this PR.

**Pull request closed without merging**